service/worker/parentclosepolicy/workflow.go

```diff
@@ -110,26 +110,26 @@ func ProcessorActivity(ctx context.Context, request Request) error {
 			//no-op
 			continue
 		case shared.ParentClosePolicyTerminate:
-			err = client.TerminateWorkflowExecution(nil, &h.TerminateWorkflowExecutionRequest{
+			err = client.TerminateWorkflowExecution(nil, &types.HistoryTerminateWorkflowExecutionRequest{
```

Comment:
Do we need to convert the error back to thrift types here for now? The error handling logic below only checks for thrift types I think.

Reply:
Yes, good catch!